pkg/images/enricher/enricher_impl_test.go

```diff
@@ -1455,6 +1458,74 @@ func TestUpdateImageFromDatabase_Metadata(t *testing.T) {
 	protoassert.Equal(t, metadata, img.GetMetadata())
 }
 
+func TestMetadataUpToDate(t *testing.T) {
+	t.Run("metadata not up to date if is nil", func(t *testing.T) {
+		e := &enricherImpl{}
+		assert.False(t, e.metadataUpToDate(nil))
+		assert.False(t, e.metadataUpToDate(&storage.Image{}))
+	})
+
+	t.Run("metadata not up to date if datasource is not found", func(t *testing.T) {
+		ctrl := gomock.NewController(t)
+
+		registrySet := registryMocks.NewMockSet(ctrl)
+		registrySet.EXPECT().Get(gomock.Any()).Return(nil)
```

Comment:
even though this won't be called, without knowing that, it seems like this test may also fail because there were no registries found. Can we put a fake one here so we know for sure this is due to the lack of dataSource?